app/build.gradle

```diff
@@ -83,6 +83,14 @@ android {
         full {
             dimension 'license'
         }
+        dw {
+            dimension 'testing'
+            applicationId 'com.dw.ooniprobe'
+            resValue "string", "APP_NAME", "DW Media Probe"
+            buildConfigField 'String', 'BASE_SOFTWARE_NAME', '"dw-media-probe-android"'
+            versionName '0.0.1'
```

Comment:
In terms of maintainability, we should figure out before hand a way to map the dw-flavoured-app version to the actual ooniprobe-android version. The simplest scheme that comes to mind is that of using the same version number and the same version code we use for ooniprobe-android. Because using the same version number may be odd (would it be odd?), perhaps we can just share the version code? What do you think?